h/migrations/versions/1ef80156ee4_add_sidebar_tutorial_dismissed_column_.py

```diff
@@ -0,0 +1,28 @@
+"""Add the sidebar_tutorial_dismissed column to user table.
+
+Revision ID: 1ef80156ee4
+Revises: 43645baa68b2
+Create Date: 2015-12-21 18:49:15.688177
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '1ef80156ee4'
+down_revision = '43645baa68b2'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    with op.batch_alter_table('user') as batch_op:
+        batch_op.add_column(sa.Column('sidebar_tutorial_dismissed',
+                                      sa.Boolean,
+                                      nullable=False,
+                                      default=False,
+                                      server_default=sa.sql.expression.true()))
```

Comment:
That's not how `server_default` works. `server_default` sets the schema for the table to include a default, so this migration will actually create a table with a schema that doesn't match the model.

But there is another issue with this, which is that adding a column with a `DEFAULT` will lock the table and write every row. So:

1. Get rid of `op.batch_alter_table` -- it's not necessary for Postgres and was only a workaround for the fact that SQLite doesn't support `ALTER TABLE ... ADD COLUMN`.
2. Add the column without a `default` (which is a no-op here) or a `server_default`.
3. Alter the column to set the (correct) `server_default`: `op.alter_column('user', 'sidebar_tutorial_dismissed', server_default=sa.sql.expression.false())`.
4. Add a second migration to update the data for preexisting users.